### ISSUE TITLE
D region sequence is empty on exportClones output

### ISSUE THREAD

**Issue by BioinfoUSAL** [milaboratory/mixcr#543]
The exportClones utility gives us empty fields about D region sequence. For example, we can not find information about the following fields: aaMutationsDRegion, nMutationsDRegion, nSeqImputedDRegion, nSeqDRegion and mutationsDetailedInDRegion.

I have used MiXCR v3.0.5 and the summary provides a valid allDHitsWithScore, bestDAlignment and dBestIdentityPercent.

Thanks for your help.

**Comment by PoslavskySV** [comment]
Did you use MiXCR v3.0.5 on the same sample and it produced D hits?

**Comment by BioinfoUSAL** [comment]
Yes, we used it on the same sample and we got a D hit on IGHD3-10*00. I can also see the alignments against IGHD3-10*00 with exportAlignments. But we can not get D region sequences on the exportClones output.

**Comment by PoslavskySV** [comment]
I see. Could you please share the _exact_ commands you used to run the analysis and the report files? Additionally, it would be very helpful if you can share the problematic data (we'll keep it confidential) - you can contact via PoslavskySV@milaboratory.com
